src/runtime/types.ts

```diff
@@ -53,6 +53,15 @@ export interface SessionDataObject {
     | SessionDataObject
 }
 
+/**
+ * Type of data returned from sign up functiom if preventLoginFlow is true
+*/
+export type SignUpResponse = {
```

Comment:
I think we should allow the passing of a generic type to the `signUp` function and then use that to type the response! 

This way we avoid hard-coding a type that may not fit to others `signUp` function and would allow them to pass a custom type!